### PR TITLE
[#1438] README.md: add build instructions

### LINKED ISSUE #1438: build instructions
Please add build instructions.

### PR DESCRIPTION
**Pull request by fyrchik**
Close #1438.

Signed-off-by: Evgenii Stratonikov <evgeniy@nspcc.ru>

### CODE REVIEW

**Review by carpawell — COMMENTED** [review]
A few words about `docker/` prefix that could build everything in the docker without `go` installed?

File: README.md [hunk 1]
```diff
@@ -46,6 +46,27 @@ platforms will be officially supported after release `1.0`.
 The latest version of neofs-node works with neofs-contract 
 [v0.13.0](https://github.com/nspcc-dev/neofs-contract/releases/tag/v0.13.0).
 
+# Building
+
+To make all binaries you need Go 1.17+ and `make`:
+```
+make build
```
Comment (carpawell) [inline README.md:53]: we do not have that target, mb `make all`?

**Review by carpawell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fyrchik) [reply README.md:53]: Fixed.

**Review by cthulhu-rider — APPROVED** [review]
LGTM

**Review by carpawell — APPROVED** [review]

**Pull request merged**